docs/user-guide/index.md

```diff
@@ -2,7 +2,7 @@
 
 ## Signup
 
-For all signup options the Name field cannot currently be changed later.
+!!! warning "For all signup options the Name field cannot currently be changed later."
```

Comment:
As someone who's had to change their name a lot of places and gotten frustrated when I couldn't, would love to change this eventually! For now the warning is a nice touch.

Reply:
Yeah this is one of those things we've just been putting off in favour of other features.  I imagine it will be added to the user settings before we roll it out more widely / soon after :)